stacks/analytics-ingest-lambda.yml

```diff
@@ -319,3 +384,51 @@ Resources:
       Dimensions:
         - Name: FunctionName
           Value: !Ref AnalyticsRedisLambdaFunction
+  AnalyticsDynamodbErrorAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[AnalyticsLambda][DynamoDB][Errors] ${EnvironmentType} > 0"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription:
+        Too many analytics dynamodb lambda errors
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 1
+      MetricName: Errors
+      Namespace: AWS/Lambda
+      Period: 300
+      Statistic: Sum
+      Threshold: 0
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: FunctionName
+          Value: !Ref AnalyticsDynamodbLambdaFunction
+  AnalyticsDynamodbIteratorAgeAlarm:
+    Type: "AWS::CloudWatch::Alarm"
```

Comment:
Also thinking maybe a warn on these getting some minutes behind (15?) wouldn't be a bad idea.